docs/DeveloperGuide.md

```diff
@@ -116,19 +116,23 @@ How the parsing works:
 ### Model component
 **API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
+<img src="images/ModelClassDiagram.png" width="600" />
 
 
 The `Model` component,
 
 * stores the address book data i.e., all `Elderly` objects (which are contained in a `UniqueElderlyList` object).
+* implicitly also stores `Nok` objects which is contained in `Elderly` objects
```

Comment:
Great this is good enough to explain the NoK implementation! Perhaps in the future we can also include a section on the purpose of `Person`.